packages/patternfly-4/react-table/src/components/Table/Body.tsx

```diff
@@ -115,32 +110,23 @@ class ContextBody extends React.Component {
   }
 }
 
-ContextBody.propTypes = {
-  className: PropTypes.string,
-  children: PropTypes.node,
-  headerData: PropTypes.array,
-  rows: PropTypes.array,
-  rowKey: PropTypes.oneOfType([PropTypes.number, PropTypes.string]).isRequired,
-  onRowClick: PropTypes.func,
-  onRow: PropTypes.func
-};
-
-ContextBody.defaultProps = {
-  className: '',
-  children: null,
-  headerData: [],
-  rows: [],
-  onRowClick: () => undefined,
-  onRow: () => undefined
-};
-
-const TableBody = props => (
+export const TableBody = ({
+  onRow = (...args:any) => Object,
+  className = '' as string,
+  children = null as React.ReactNode,
+  rowKey = 'id' as string,
+  onRowClick = (...args: any) => undefined as any,
+  ...props
+}: TableBodyProps) => (
   <TableContext.Consumer>
-    {({ headerData, rows }) => <ContextBody headerData={headerData} rows={rows} {...props} />}
+    {({ headerData, rows }) => 
+      <ContextBody 
+        headerData={headerData} 
```

Comment:
I think that rest of props are missing in here.

Reply:
thanks! added `...rest` for Context props as well (so headerRows will be passed in): https://github.com/patternfly/patternfly-react/pull/2360/commits/fe06c8dc05fff246931db441ad24cf306ff742e9